test/controllers/asset/noneeabusiness/add/AnswersControllerSpec.scala

```diff
@@ -145,5 +146,24 @@ class AnswersControllerSpec extends SpecBase with MockitoSugar with ScalaFutures
       application.stop()
     }
 
+    "return an internal server error given None is returned from NonEeaBusinessAssetMapper" in {
```

Comment:
Were did this new test come from?

Reply:
just a bit of low hanging fruit for increasing the coverage percentage